scripts/check_deployment_secrets.py

```diff
@@ -15,17 +13,15 @@
 
 #1
 For the json returned from the secret in GOOGLE_APPLICATION_SECRETS_SECRETS_NAME:
-    `auth_uri` should be in ['https://auth.data.humancellatlas.org/authorize',
-                             'https://auth.dev.data.humancellatlas.org/authorize']
-    `token_uri` should be in ['https://auth.data.humancellatlas.org/oauth/token',
-                              'https://auth.dev.data.humancellatlas.org/oauth/token']
+    `auth_uri` should be 'https://accounts.google.com/o/oauth2/auth'
+    `token_uri` should be 'https://oauth2.googleapis.com/token'
```

Comment:
This should probably be `auth.ucsc.ucsc-cgp-redwood.org`?

Reply:
Yes, this should be changed. 
although we should decide if we want to use `ucsc` as the stage name instead of `dev`....

Reply:
Oops, it's actually set to `auth.ucsc.ucsc-cgp-redwood.org` in the script, this is just a docstring.